mix.exs

```diff
@@ -25,7 +25,12 @@ defmodule Tirexs.Mixfile do
   end
 
   defp deps do
-    [ {:exjsx, "~> 3.2.0"}, {:ex_doc, "~> 0.12", only: :dev}, {:earmark, "~> 1.0", only: :dev} ]
+    [
+      {:exjsx, "~> 3.2.0"},
+      {:aws_auth, "~> 0.5.1"},
```

Comment:
this lib must be maybe at dev, if i don't use AWS, why this lib comes and compiles? i can add it manual, so include it just for dev and test